docker-compose.yml

```diff
@@ -41,7 +41,7 @@ services:
     ports:
       - "5432:5432"
     environment:
-      POSTGRES_PASSWORD: example
+      POSTGRES_PASSWORD: postgres
```

Comment:
How does the DB password change help?

Reply:
It simplifies reproducing: running lookout with docker docs with style-analyzer quickstart docs does not require any changes.